### DISCUSSION TITLE
Secure dashboard with username and password

### DISCUSSION THREAD

**Discussion by billyjoebob999** [Koenkk/zigbee2mqtt#17297] [General]
Looked through the documentation but couldn't find if this is supported. I have Z2M setup in a docker and would like to have it prompt for logon before you can access the dashboard for a little extra security. Is it possible to secure Z2M with a username/password? 

Edit: Just to be clear I'm talking about requiring a username/password on the frontend (web gui).

**Comment by junosuarez** [comment]
You can configure an auth token, see https://www.zigbee2mqtt.io/guide/configuration/frontend.html#advanced-configuration

**Reply by billyjoebob999** [reply]
Thank you! I was able to get this setup pretty easily. One thing I noticed is the auth token seems to get cached no longer requiring you enter unless you clear the browser cache. Is it possible to force a reentry every time a new browser session is opened?
